client/views.py

```diff
@@ -58,6 +58,7 @@ def homepage(request):
             "total_help_requests": HelpRequest.objects.count() + 84
         }
     }
+    #added 1786 and 84 since those are the stats for before this app
```

Comment:
Comments should be right before the lines being explained - when reading code, you should understand a line when reading it, and not later.